src/ui/components/Header/ShareDropdown.js

```diff
@@ -27,32 +27,30 @@ const GET_RECORDING_PRIVACY = gql`
   }
 `;
 
-const GET_OWNER_AUTH_ID = gql`
-  query GetOwnerAuthId($recordingId: uuid!) {
+const GET_OWNER_USER_ID = gql`
+  query GetOwnerUserId($recordingId: uuid!) {
     recordings(where: { id: { _eq: $recordingId } }) {
-      user {
-        auth_id
-      }
+      user_id
     }
   }
 `;
 
 function useIsOwner(recordingId) {
-  const { user, isAuthenticated } = useAuth0();
-  const { data, loading } = useQuery(GET_OWNER_AUTH_ID, {
+  const { userId } = useToken() || {};
```

Comment:
could `useToken` always return an object to be safer?

Reply:
Right now `undefined` means auth0 is still loading, whereas `{}` means the user didn't try to authenticate (and `{ error: ... }` means authentication failed), but that could be changed of course.